qurator/dinglehopper/tests/test_integ_ocrd_cli.py

```diff
@@ -13,6 +14,7 @@
 data_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="only on unix")
```

Comment:
Does the OCR-D interface really not work on win32? @kba, do you have any insight on this?

Reply:
The reason is related to the `AtomicWriterPerms` which uses `chmod` features only available on unix systems.

See also OCR-D/core#636.

The change in the mentioned pull requests allows the development on windows but not the execution of the functionality =)

Reply:
> @kba, do you have any insight on this?

Theoretically, most of OCR-D/core should work on Windows, but it is largely untested and we don't recommend it. But if developers like @b2m notice these platform-dependent constructs, we'll try to fix them.